### PR TITLE
Chemical Wiki Descriptions, Recipe Tweaks & Effect Tweaks

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
o Changes almost all wordings in the chemical wiki section to have up-to-date, consistent and relevant information.
o Tweaked chemical recipes, namely:
- All plasma catalysts were reduced from 5u to 1u
- Epinephrine no longer requires diethylamine, creates 5u epinephrine
- Dexalin requires 2u hydrogen,1u oxygen, 1u plasma catalyst, creates 3u dexalin
- Lexorin requires 1u heartbreaker toxin, 1u fluorine, 1u plasma, creates 3u lexorin
- Ultravasculine requires 1u dylovene, 1u sodium, 1u plasma catalyst
- Sigynate requires 1u iodine, 1u kelotane, 1u sugar, creates 3u sigynate
- Phenol requires 1u oxygen, 1u hydrogen, 1u carbon
- Diphenymethylamine requires 1u ethyloxyephedrine, 1u phenol, 1u sodium carbonate, makes 3u diphenylmethylamine
- Ammonia reaction creates 3u instead of 4u
- Sodium carbonate no longer requires ammonia, creates 3u sodium carbonate

o  Tweaked chemical effects, namely:
- Bicaridine no longer causes drunkness
- Dermaline no longer causes drunkness on slight overdose
- Nocturine no longer causes poison damage on overdose (60u to deal 2 poison per second are you real)
- Sigynate cures 2 caustic a second instead of 6

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Alekshhh
- tweak: The chemical page of the wiki has been heavily reworded, and a few chemicals had their recipes tweaked and effects changed a little

### CODE REVIEW

**Comment by Interrobang01** [comment]
Great PR. This has been needed for a while now, thank you. I don't have anything negative to comment on, everything here is good.

...fantastic licoxide description btw

**Comment by Alekshhh** [comment]
if anyone has suggestions for desc recipes or effects lemme know since I already have it open

File: Resources/Locale/en-US/reagents/meta/medicine.ftl [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-reagent-desc-dermaline = An advanced chemical that is more effective at treating burn damage than Kelotane.
+reagent-desc-dermaline = An advanced chemical that is more effective at treating burn damage than kelotane.
 
 reagent-name-dexalin = dexalin
-reagent-desc-dexalin = Used for treating oxygen deprivation. In most cases where it is likely to be needed, the strength of Dexalin Plus will probably be more useful.
+reagent-desc-dexalin = Used for treating minor oxygen deprivation. A required reagent for dexalin plus.
 
 reagent-name-dexalin-plus = dexalin plus
-reagent-desc-dexalin-plus = Used in treatment of extreme cases of oxygen deprivation. Effective at healing blood loss damage.
+reagent-desc-dexalin-plus = Used in treatment of extreme cases of oxygen deprivation and bloodloss. Flushes lexorin out the blood stream.
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/medicine.ftl:26]: out of

File: Resources/Locale/en-US/reagents/meta/medicine.ftl [hunk 2]
```diff
@@ -1,56 +1,56 @@
 reagent-name-cryptobiolin = cryptobiolin
-reagent-desc-cryptobiolin = Causes confusion and dizziness. This is essential to make Spaceacillin.
+reagent-desc-cryptobiolin = Causes confusion and dizziness. An essential reagent to make spaceacillin.
 
 reagent-name-dylovene = dylovene
-reagent-desc-dylovene = A broad-spectrum anti-toxin, which treats toxin damage in the blood stream. Overdosing will cause vomiting, dizzyness and pain.
+reagent-desc-dylovene = A broad spectrum anti-toxin, which treats toxin damage in the blood stream. Overdosing will cause vomiting, dizzyness and pain.
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/medicine.ftl:5]: broad-spectrum was correct

File: Resources/Locale/en-US/reagents/meta/medicine.ftl [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
-reagent-desc-dexalin = Used for treating oxygen deprivation. In most cases where it is likely to be needed, the strength of Dexalin Plus will probably be more useful.
+reagent-desc-dexalin = Used for treating minor oxygen deprivation. A required reagent for dexalin plus.
 
 reagent-name-dexalin-plus = dexalin plus
-reagent-desc-dexalin-plus = Used in treatment of extreme cases of oxygen deprivation. Effective at healing blood loss damage.
+reagent-desc-dexalin-plus = Used in treatment of extreme cases of oxygen deprivation and bloodloss. Flushes lexorin out the blood stream.
 
 reagent-name-epinephrine = epinephrine
-reagent-desc-epinephrine = Effective at bringing people back from a critical state. Reduces some stun times. Easy to overdose on.
+reagent-desc-epinephrine = An effective stabilizing chemical used to keep a critical person from dieing to asphyxiation while patching up minor damage during crit. Flushes lexorin out the blood stream at the cost of more epinephrine, but may add histamine. Helps reduce stun time. Commonly found in the form of emergency medipens.
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/medicine.ftl:29]: dying

File: Resources/Locale/en-US/reagents/meta/medicine.ftl [hunk 4]
```diff
@@ -1,56 +1,56 @@
...
-reagent-desc-hyronalin = A weak treatment for radiation damage. Considered to be useful mainly for genetic modification, where it reduces radiation levels, and thus the chance of genetic mutations. Largely outclassed by Arithrazine.
+reagent-desc-hyronalin = A weak treatment for radiation damage. A precursor to arithrazine and phalanximine. Can cause vomitting.
 
 reagent-name-ipecac = ipecac
-reagent-desc-ipecac = Induces vomiting. Useful for stopping a poisoning that isn't done metabolizing.
+reagent-desc-ipecac = A rapid-acting emetic. Useful for stopping unmetabolized poisons, or mass-vomiting sessions.
 
 reagent-name-inaprovaline = inaprovaline
-reagent-desc-inaprovaline = Inaprovaline is a synaptic stimulant and cardiostimulant. Commonly used to stabilize patients- it stops oxygen loss when the patient is in critical health. It'll also slow down bleeding by a good amount. Acts as a decent painkiller.
+reagent-desc-inaprovaline = Inaprovaline is a synaptic stimulant and cardiostimulant, commonly used to treat asphyxiation damage caused during critical states and reduce bleeding. Used in many advanced medicine.
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/medicine.ftl:38]: medicines

File: Resources/Locale/en-US/reagents/meta/medicine.ftl [hunk 5]
```diff
@@ -1,56 +1,56 @@
...
 
 reagent-name-kelotane = kelotane
-reagent-desc-kelotane = Treats burn damage and prevents infection.
+reagent-desc-kelotane = A weak yet simple and effective medicine used to aid in treating minor burn damage.
 
 reagent-name-leporazine = leporazine
-reagent-desc-leporazine = This keeps a patient's body temperature stable. High doses can allow short periods of unprotected EVA, but prevents use of the cryogenics tubes.
+reagent-desc-leporazine = A chemical used to stabilize body temperature and rapidly cure cold damage. Great for unprotected EVA travel, but prevents use of cryogenic tubes.
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/medicine.ftl:44]: extra vehicular activities travel pro

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
> 
> o Tweaked chemical effects, namely:
> 
>     * Bicaridine no longer causes drunkness
> 
>     * Dermaline no longer causes drunkness on slight overdose

keep both of these

**Comment by AJCM-git** [comment]
@Alekshhh You coming back to this? (also you committed the engine)

**Comment by Alekshhh** [comment]
FUCK

**Comment by Alekshhh** [comment]
I'll prob just make a new one with only the wiki tweaks and recipe tweaks

**Pull request closed without merging**